### ISSUE TITLE
could not find `regex` in the list of imported crates

### ISSUE THREAD

**Issue by rbox-risk** [yassun7010/serde_valid#86]
Hi thank you for great crate! I have complex validations (on struct fields and on the overall struct) and this works great so far.

Regarding use of `ValidatePattern` 

```
pub struct MYStruct {
//other_fields
    #[validate(max_length = 155)]
    #[validate(pattern = r"^[^!>]*$")]
    pub Some_Field: Option<String>,
}
```

I get the below error when using it on one of my fields. Of course, I can `cargo add regex` - but I did not find mention of this either in feature flag or in the docs.

```
error[E0433]: failed to resolve: could not find `regex` in the list of imported crates
  --> src\types.rs:15:23
   |
15 | #[derive(Deserialize, Validate)]
   |                       ^^^^^^^^ could not find `regex` in the list of imported crates
   |
   = note: this error originates in the derive macro `Validate` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by yassun7010** [comment]
@rbox-risk 

This is probably due to a version mismatch with the regex used by serde_valid.

I exported regex from serde_valid, so this problem has been resolved in [v0.25.1](https://github.com/yassun7010/serde_valid/tree/v0.25.1).

**Comment by yassun7010** [comment]
I just released v1.0.4.

If you are not using the old features, you may want to switch to that.
